main.go

```diff
@@ -99,20 +102,20 @@ func main() {
 			assumedSessionToken := *assumedRole.Credentials.SessionToken
 
 			for _, region := range supportedRegions {
-				fmt.Println("\n Current Region : ", region)
+				logger = log.WithValues("AccountName", account.Name, "AccountID", account.Spec.AwsAccountID, "Region", region)
```

Comment:
I think there needs to be more context here

Reply:
All this line is doing is updating the logger to always display the region when logging. No longer a need to display the region being processed beforehand

Reply:
I think its still helpful to say "Cleaning region y"